crates/fluvio-version-manager/src/common/mod.rs

```diff
@@ -6,6 +7,12 @@ use std::path::PathBuf;
 
 use anyhow::{Error, Result};
 
+/// The Target Architecture of the current build (e.g. "aarch64-apple-darwin")
+///
+/// This is injected by Cargo during the build process. Refer to `build.rs` for
+/// more information.
+pub const TARGET: &str = env!("TARGET");
```

Comment:
Same issue.  These config items should be passed as parameters from CLI env var not from const

Reply:
The purpose of this is to gather the target triple to use for artifacts download.
This is not an env var that can be set at runtime, but instead is build time.

This is later used when fetching the registry to retrieve the compatible artifacts.
We could have an environment variable to overwrite this.